mqt/benchviewer/src/backend.py

```diff
@@ -1,11 +1,15 @@
-import pandas as pd
+import io
 import os
 import re
-import json
-import io
-from zipfile import ZipFile, ZIP_DEFLATED
 from pathlib import Path
 from typing import List, Iterable
+from zipfile import ZipFile, ZIP_DEFLATED
+
+import pandas as pd
+import requests
+from importlib import metadata
```

Comment:
`importlib.metadata` is only available since Python 3.8 and is no longer provisional since Python 3.10 (see https://docs.python.org/3/library/importlib.metadata.html). The backport package `importlib_metadata` is required for older Python versions. Thus, the following should be adopted
```suggestion
try:
    from importlib import metadata
except ImportError:
    import importlib_metadata as metadata
```
And the following should be added to the requirements
```
"importlib_metadata>=3.6; python_version < '3.10'"
```

Reply:
done